apps/emqx/src/emqx_schema.erl

```diff
@@ -1079,65 +1158,114 @@ fields("deflate_opts") ->
             sc(
                 range(8, 15),
-                #{default => 15}
+                #{
+                    default => 15,
+                    desc => "Specifies the size of the compression context for the server."
+                }
             )},
         {"client_max_window_bits",
             sc(
                 range(8, 15),
-                #{default => 15}
+                #{
+                    default => 15,
+                    desc => "Specifies the size of the compression context for the client."
+                }
             )}
     ];
 fields("broker") ->
     [
         {"enable_session_registry",
             sc(
                 boolean(),
-                #{default => true}
+                #{
+                    default => true,
+                    desc => "Enable session registry"
+                }
             )},
         {"session_locking_strategy",
             sc(
                 hoconsc:enum([local, leader, quorum, all]),
-                #{default => quorum}
+                #{
+                    default => quorum,
+                    desc =>
+                        "Session locking strategy in a cluster.<br/>\n"
+                        " - `local`: only lock the session on the current node\n"
+                        " - `one`: select only one remove node to lock the session\n"
+                        " - `quorum`: select some nodes to lock the session\n"
+                        " - `all`: lock the session on all the nodes in the cluster"
+                }
             )},
         {"shared_subscription_strategy",
             sc(
                 hoconsc:enum([random, round_robin, sticky, hash_topic, hash_clientid]),
-                #{default => round_robin}
+                #{
+                    default => round_robin,
+                    desc =>
+                        "Dispatch strategy for shared subscription.<br/>\n"
+                        " - `random`: dispatch the message to a random selected subscriber\n"
+                        " - `round_robin`: select the subscribers in a round-robin manner\n"
+                        " - `sticky`: always use the last selected subscriber to dispatch,\n"
+                        "   until the subscriber disconnected.\n"
+                        " - `hash`: select the subscribers by the hash of `clientIds`"
+                }
             )},
         {"shared_dispatch_ack_enabled",
             sc(
                 boolean(),
-                #{default => false}
+                #{
+                    default => false,
+                    desc =>
+                        "Enable/disable shared dispatch acknowledgement for QoS1 and QoS2 messages.<br/>\n"
+                        " This should allow messages to be dispatched to a different subscriber in\n"
+                        " the group in case the picked (based on `shared_subscription_strategy`) subscriber\n"
+                        " is offline."
```

Comment:
For my own information: this re-routing to an online subscriber can happen when this parameter is enabled?

Reply:
Not sure about the correct answer: I copied that from `emqx.conf.all`